packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -8,6 +9,7 @@ export interface TableFieldOptions {
   align: FieldTextAlignment;
   displayMode: TableCellDisplayMode;
   hidden?: boolean;
+  render?: (props: TableCellProps) => React.ReactNode | null | number | string;
```

Comment:
Don' think we need this as this type is not used anywhere in the end.

Reply:
Thanks mate, good catch 👍